math/libfaiss/Portfile

```diff
@@ -24,39 +22,134 @@ long_description    Library for efficient similarity search and clustering of \
                     with complete wrappers for Python/numpy. Some of the most \
                     useful algorithms are implemented on the GPU.
 
+checksums           rmd160  3b1180dc7ceadc484bb86cfe32fb5cd1c4a694b9 \
+                    sha256  6962641a9183e12d394aa2f9c443593734a373d60c61fd0633edda5029d2fa78 \
+                    size    2762193
+
 compilers.choose    cxx
 # The Apple native llvm compiler does not support OpenMP.
 compiler.blacklist  llvm-gcc-4.2
-compiler.fallback   macports-gcc-7
+compiler.fallback   macports-gcc-9
+configure.cxx_stdlib libstdc++
 compilers.setup
 
-use_configure       no
+depends_lib-append  port:libomp
+
+configure.args      --without-cuda
+
+build.env-append    CC=${configure.cxx} \
+                    "CFLAGS=-fPIC -m64 -Wall -g -O3 -msse4 -mpopcnt -fopenmp \
+                        -Wno-sign-compare -std=c++11 -I/usr/include/malloc/" \
```

Comment:
you're adding `-std=c++11` here so you likely want to make sure you get a compatible compiler (i.e., use `compiler.cxx_standard   2011`)

Reply:
Added thanks.